### PR TITLE
More helpers and helper tests

### PR DESCRIPTION
**Pull request by infinitewarp**
This is in support of #33 and #72, but since the actual report testing is taking a while, I wanted to go ahead and get some eyes on these new helpers.

### CODE REVIEW

File: cloudigrade/account/tests/helper.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        region = random.choice(helper.SOME_AWS_REGIONS)
+
+    return Instance(
+        account=account,
+        ec2_instance_id=ec2_instance_id,
+        region=region
+    )
+
+
+def generate_instance_events(instance, powered_times, ec2_ami_id=None,
```
Comment (adberglund) [inline cloudigrade/account/tests/helper.py]: `powered_times` here referred to as `powerd_on_times` in docstring.

File: cloudigrade/account/tests/test_helper.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+from account.models import Account, Instance, InstanceEvent
+from account.tests import helper
+from util.tests import helper as util_helper
+
+
+class AccountHelperTest(TestCase):
+    """Test helper functions test case."""
+
+    def test_generate_generate_account_default(self):
```
Comment (adberglund) [inline cloudigrade/account/tests/test_helper.py]: is the double `generate` here and in fhte next test intentional?

File: cloudigrade/account/tests/helper.py (hunk 1 above)
Comment (infinitewarp) [reply cloudigrade/account/tests/helper.py]: good 👀.

**Pull request merged**